job/server/src/test/java/alluxio/job/plan/transform/format/JobPathTest.java

```diff
@@ -15,76 +15,78 @@
 import static org.junit.Assert.assertNotEquals;
 import static org.mockito.Mockito.any;
 import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.mockStatic;
 import static org.mockito.Mockito.times;
-import static org.powermock.api.mockito.PowerMockito.mockStatic;
-import static org.powermock.api.mockito.PowerMockito.verifyStatic;
-import static org.powermock.api.mockito.PowerMockito.when;
+import static org.mockito.Mockito.verify;
 
 import alluxio.client.ReadType;
 import alluxio.conf.PropertyKey;
 
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.FileSystem;
 import org.apache.hadoop.security.UserGroupInformation;
+import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.modules.junit4.PowerMockRunner;
+import org.mockito.MockedStatic;
 
-@RunWith(PowerMockRunner.class)
-@PrepareForTest({JobPath.class, UserGroupInformation.class})
 public class JobPathTest {
 
+  private MockedStatic<UserGroupInformation> mMockedStaticUserGroupInfo;
+
   @Before
   public void before() throws Exception {
-    mockStatic(UserGroupInformation.class);
+    mMockedStaticUserGroupInfo = mockStatic(UserGroupInformation.class);
+    mMockedStaticUserGroupInfo.when(UserGroupInformation::getCurrentUser).thenReturn(null);
```

Comment:
Since this mock is only used in `testCache` test, can you remove this field and initialize it along with `jobPathMocked` in the try-with-resources construct?

Reply:
done~